MQ2PluginManager.cpp

```diff
@@ -4,30 +4,37 @@
 	around until things worked. There may be some residual code that is not even required by the
 	plugin in here, but if it is, I'm unaware of it. Chances are some of these things I just deleted
 	entirely and tried to run after a succesful compile only to crash and therefore was readded.
+
+	July  9, 2024 Grimmier  -- Added ImGui Window to the mqsettings Panel.
+	July 30, 2024 Grimmier  -- Removed the in game XML window and replaced with a IMGUI window.
+							-- Added a toggle button to the mqsettings panel window.
+							-- Switched the /pluginman command to toggle the IMGUI window instead of the old XML.
 */
 
 #include <mq/Plugin.h>
 #include "CPluginTree.h"
-#include "CPluginToolWnd.h"
+#include "imgui/ImGuiUtils.h"
+#include <imgui_internal.h>
 
 PreSetup("MQ2PluginManager");
 PLUGIN_VERSION(2019.0828);
 
 //=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=//
-CPluginToolWnd* PluginWnd = nullptr;
 CPluginTree* PluginTree = nullptr;
 std::unordered_set<std::string> LoadedPlugins;
 //=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=//
 
 static void DrawPluginManager_MQSettingsPanel();
 static void PluginManagerCommand(PlayerClient*, const char*);
+static bool s_showWindow = false;
+static void ImGui_ToggleWindow();
 
 //=-=-=-=-=- Plugin members =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=//
 PLUGIN_API void InitializePlugin()
 {
-	WriteChatColor("\aoLoading MQ2PluginManager v0.0.2 ...");
+	WriteChatColor("\aoLoading MQ2PluginManager v0.0.3 ...");
```

Comment:
if we're going to leave the version on it here, we should put the version somewhere and then use it here

Reply:
dropped it since it really doesn't do anything